test/gun_papas_cfg.py

```diff
@@ -3,6 +3,8 @@
 import heppy.framework.config as cfg
 
 import logging
+
+import sys
```

Comment:
can you remove this import if the sys module is not used? 
if it's used in the main section only, the import should go there.